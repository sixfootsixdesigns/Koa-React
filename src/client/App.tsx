import React from 'react';
import './app.scss';
import { RouteComponentProps } from '@reach/router';

export class App extends React.Component<RouteComponentProps> {
  public render() {
    return <div>Hello this is a react app</div>;
  }
}
