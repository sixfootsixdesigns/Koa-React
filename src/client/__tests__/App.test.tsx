import React from 'react';
import { render, cleanup } from 'react-testing-library';
import { App } from '../App';

afterEach(() => {
  cleanup();
});

describe('App', () => {
  it('Renders page]', () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
