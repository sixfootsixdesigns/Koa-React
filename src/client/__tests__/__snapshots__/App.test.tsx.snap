// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders page] 1`] = `
<div>
  Hello this is a react app
</div>
`;
